src/fragments/lib/logging/common/flush_logs/flush_logs.mdx

```diff
@@ -0,0 +1,33 @@
+When using the Amplify Logger, all logged messages are saved locally on the user's device first and then flushed at a set interval that you can [customize](#change-automatic-log-flush-interval). You can also choose to flush events manually if needed by following the steps outlined in this section.
+
+import ios0 from '/src/fragments/lib/logging/ios/flush_logs/flush_logs.mdx';
+import android0 from '/src/fragments/lib/logging/android/flush_logs/flush_logs.mdx';
+
+<Fragments
+  fragments={{
+    ios: ios0,
+    android: android0
+  }}
+/>
+
+## Change automatic log flush interval
+To change the time interval when logs are automatically flushed and sent to CloudWatch.
```

Comment:
You can customize the time internal for when logs are automatically flushed and sent to CloudWatch.

Reply:
Updated per suggestion.